spec/models/organization_spec.rb

```diff
@@ -407,7 +406,7 @@
 
     context "when the name or profile_image has not changed or the organization has no articles" do
       it "does not call SocialImageWorker.perform_async" do
-        expect(Images::SocialImageWorker).not_to receive(:perform_async)
+        expect(Images::SocialImageWorker).not_to have_received(:perform_async)
```

Comment:
I suppose it makes sense to `have_received` after `organization.save` (in contrast to `receive` that was previously here)

Reply:
Yeah I agree, the linter made me do it but it also makes sense and I was also like "sure, I suppose" :-D